rpc/core/1

```diff
@@ -1,55 +0,0 @@
-package core
```

Comment:
yes, why this was committed in the first place I wondered about today as well :)